content/showcase.en.md

```diff
@@ -3,5 +3,6 @@ title: Showcase
 hero:
   title: Showcase
   subtitle: Discover products built with OpenFisca
+previous_reuses: Previous reuses
```

Comment:
We don't really mention “reuse” in the showcase, we talk more of “products”, so let's stay consistent 🙂 
```suggestion
previous_reuses: Disused products
```